llama_index/embeddings/openai.py

```diff
@@ -224,23 +224,33 @@ def __init__(
         self.deployment_name = deployment_name
         self.query_engine = get_engine(mode, model, _QUERY_MODE_MODEL_DICT)
         self.text_engine = get_engine(mode, model, _TEXT_MODE_MODEL_DICT)
+        self._extra_params = kwargs
```

Comment:
let's name these openai_kwargs and not pass to base class. reason is i'm not sure what the best interface is for passing arbitrary kwargs quite yet. will avoid unexpected behavior with other llm providers